### DISCUSSION TITLE
I can't add the Aqara FP1

### DISCUSSION THREAD

**Discussion by ykuijs** [Koenkk/zigbee2mqtt#15317] [Devices]
A few months back I bought an Aqara Presence Sensor FP1 and I am trying to add that to my Home Assistant instance via Zigbee2MQTT. Unfortunately, without luck so far. The Debug log shows:
```
info 2022-09-13 23:29:31: Starting interview of '0x54ef4410005457ff'
info 2022-09-13 23:29:31: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x54ef4410005457ff","ieee_address":"0x54ef4410005457ff","status":"started"},"type":"device_interview"}'
error 2022-09-13 23:31:49: Failed to interview '0x54ef4410005457ff', device has not successfully been paired
info 2022-09-13 23:31:49: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x54ef4410005457ff","ieee_address":"0x54ef4410005457ff","status":"failed"},"type":"device_interview"}'
```

Zigbee2MQTT keeps saying the device is unsupported. I am running a recent build of Zigbee2MQTT and am using the Sonoff Zigbee USB stick with the most recent custom firmware. My HA instance is running in a Proxmox VM and I have mapped the USB stick to the HA VM. I have set Zigbee2MQTT to use channel 25.

I recently purchased an Aqara Hub E1 to test if the device actually works and if a newer firmware is available. I am able to connect the FP1 successfully to the Aqara Hub and the Aqare Home app says I am running the most recent firmware (v0054).

I created this issue back in September: #14035 

Even though the device works via the Aqara Hub, I would like to use it via Zigbee2MQTT, so I do not have to start using an additional Hub.

When searching the Internet for a solution I find a lot of reports stating that the devices works fine, but I have no clue why it doesn't on my end. Would appreciate it if someone could provide a little help in figuring out the root cause.

**Comment by lux73** [comment]
i have the same Problem few Weeks ago with my newly purchased FP1 - only thing that works is pairing very near Coordinator.

pairing far away from him does not work. i'm too on Channel 25. CC2652RB my Coordinator.
